Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -391,6 +398,7 @@ public class ManifestLoader: ManifestLoading {
                     "-F", manifestPath,
                     "-lPackageDescription",
                     "-D", "TUIST",
+                    "-package-description-version", packageVersion.description,
```

Comment:
nit: just to group things together
```suggestion
                    "-lPackageDescription",
                    "-package-description-version", packageVersion.description,
                    "-D", "TUIST",
```